### DISCUSSION TITLE
Unable to Access YouTube - Possible Software or Proxy Issue

### DISCUSSION THREAD

**Discussion by Halerfermatqing** [InfinityLoop1308/PipePipe#414] [Q&A] [answered]
I am writing to report an issue I am facing with the pipepipe application. I am unable to login to YouTube and would like to inquire whether this is a software problem or related to the proxy settings.

Here are the details of the issue:

- **User Action:** Requested stream
- **Request:** https://www.youtube.com/watch?v=aFsGDcy-6hc
- **Content Country:** CN
- **Content Language:** zh-CN
- **App Language:** zh_CN_#Hans
- **Service:** YouTube
- **Version:** 3.7.0
- **OS:** Linux realme/RMX3700/RE585F:14/UP1A.231005.007/T.185ec58-2e755-4f248:user/release-keys 14 - 34

The error message I received is: "Sign in to confirm that you're not a bot."

Below is the crash log for your reference:

 
org.schabi.newpipe.extractor.exceptions.ContentNotAvailableException: Got error: "Sign in to confirm that you're not a bot"
	at org.schabi.newpipe.extractor.services.youtube.extractors.YoutubeStreamExtractor.checkPlayabilityStatus(YoutubeStreamExtractor.java:1032)
	...
	at java.lang.Thread.run(Thread.java:1012)
 

Could you please assist me in determining the cause of this issue and provide a solution? I am unsure whether it is a problem with the pipepipe application itself or if it is related to my proxy settings.

Your prompt assistance in this matter would be greatly appreciated.

Thank you for your time and support.

Best regards,
Haler.

**Comment by InfinityLoop1308** [comment, marked as answer]
Hi, this is a known issue, see #415. You may need a VPN to bypass it.

**Reply by Halerfermatqing** [reply]
Thank you, I've received it.

**Comment by InfinityLoop1308** [comment]
Now you can sign in to avoid the issue in 3.8.2.
